doc/faq/virtualenv_faq.rst

```diff
@@ -0,0 +1,105 @@
+.. _virtualenv-faq:
+
+***********************************************
+Working with Matplotlib in Virtual environments
+***********************************************
+
+.. contents::
+   :backlinks: none
+
+
+.. _introduction:
+
+Introduction
+============
+
+When running :mod:`matplotlib` in a virtual environment you may discover a
+few issues. :mod:`matplotlib` itself has no issue with virtual environments.
+However, the GUI frameworks that :mod:`matplotlib` uses for interactive
+figures have some issues with virtual environments. Everything below assumes
+some familiarity with the Matplotlib backends as found in :ref:`What is a
+backend? <what-is-a-backend>`.
+
+If you only use the ``IPython/Jupyter`` inline and ``notebook/nbagg`` backends
```

Comment:
The `inline` also needs the backticks, and `notebook/nbagg` might be `notebook` or (alias) `nbagg`.  Correct?  Otherwise, I have a little trouble parsing this, and realizing that it is very specific; just using the Jupyter notebook, for example, is not enough.  This is in contrast to ordinary gui operations, in which using IPython or the notebook tends to solve all problems, regardless of backend.  (Well, most, anyway.)